namespaces/api-description-namespace.md

```diff
@@ -250,7 +250,7 @@ transitions.
             {
               "element": "member",
               "meta": {
-                  "class": ["user"]
+                  "classes": ["user"]
```

Comment:
Sometimes we don't use indentation for value of `classes` and sometimes we do (check the `api` above). I don't have a strong opinion on this. Do we want to be consistent on this or can it be left like this?